internal/pkg/service/buffer/store/file_state_test.go

```diff
@@ -0,0 +1,105 @@
+package store_test
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/benbjohnson/clock"
+	"github.com/keboola/go-client/pkg/keboola"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/config"
+	bufferDeps "github.com/keboola/keboola-as-code/internal/pkg/service/buffer/dependencies"
+	storePkg "github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/filestate"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/key"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/model/column"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/common/dependencies"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/etcdhelper"
+)
+
+func TestStore_SetFileState_Transitions(t *testing.T) {
```

Comment:
V Go je mozne testovat dvoma sposobmi:
- blackbox, ak je `package PGK_test`.
  - tj. testovany pkg sa spolu s dalsimi importne do noveho pkg, nemas pristup k private metodam/fieldom
- whitebox, ak je `package PKG`
  - mas pristup ku private, ale niekedy to moze viest ku cyklickym zavislostiam medzi balickami
  
  
Tu prave testujem vela `private` metod, ... ale niektore potrebujem testovat teraz uz ako `blackbox`, spolu so statistikami, preto som ich vyclenil do samostatnych suborov s `package store_test`.